File: 389. Find the Difference.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        ans = [str(x) for x in t]
        
        print(ans)
        for x in range(len(s)):
            ans.remove(s[x])
            
        #print(ans)
        return ans[0]
        

File: go_solution_set1/389. Find the Difference.go
func findTheDifference(s string, t string) byte {
    m := make(map[int]int)

    for _, val := range s{
        _,present := m[int(val)]
        if present{
            m[int(val)]+=1
        } else {
            m[int(val)] = 1
        }
    }

    for _, val := range t{
        _,present := m[int(val)]
        if present{
            m[int(val)]-=1
        } else {
            m[int(val)] = -1
        }
    }

    for key,val := range m{
        if val == -1{
            return byte(key)
        }
    }


    return byte(0)
}
